qa/tasks/vstart_runner.py

```diff
@@ -109,7 +109,7 @@ def respawn_in_path(lib_path, python_paths):
                 python_paths.append(g_exp)
 
     ld_path = os.path.join(os.getcwd(), "lib/")
-    print "Using guessed paths {0} {1}".format(ld_path, python_paths)
+    print("Using guessed paths {0} {1}".format(ld_path, python_paths))
```

Comment:
we cannot assume python3 yet. might want to add this in the beginning of this script.
```py
from __future__ import print_function
```

Reply:
@tchaikov I tested it in Python 2 and it worked flawlessly.
Nevertheless I'm ok with adding this line.

Reply:
@alfonsomthd sorry, i didn't realize that we require python2.7. please ignore this comment.